rules/properties-alphabetical-order/tests/index.js

```diff
@@ -68,6 +68,9 @@ testRule({
 		{
 			code: 'a { font-size: 1px; -moz-osx-font-smoothing: grayscale; -webkit-font-smoothing: antialised; font-weight: bold; }',
 		},
+		{
+			code: 'a { color: #000; span { bottom: 1em; top: 1em; } width: 25%;}',
```

Comment:
I applied your suggestion, but still... this test was supposed to test case from #59

Reply:
Under the hood it's the same AST. But now we could use standard CSS, instead of SCSS feature that almost no one uses, and I will never add support to my plugins.